v2/yarpchttp/yarpchttpfx/yarpchttpfx.go

```diff
@@ -0,0 +1,136 @@
+package yarpchttpfx
+
+import (
+	"context"
+	"net/url"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	"go.uber.org/config"
+	"go.uber.org/fx"
+	yarpc "go.uber.org/yarpc/v2"
+	"go.uber.org/yarpc/v2/yarpchttp"
+	"go.uber.org/zap"
+)
+
+const (
+	_name                     = "yarpchttpfx"
+	_inboundConfigurationKey  = "yarpchttp.inbounds"
+	_outboundConfigurationKey = "yarpchttp.outbounds"
```

Comment:
Oh, we could do `yarpc.http.outbounds` and a lot of config would not change. 🤔

Would only be a problem if you tried to use both v1 and v2 in a single appliction.

Reply:
And that problem almost seems like a good problem to have! It will prevent people from using both :)

Reply:
🤔 nothing will truly prevent people from using both.

We're not squashing on the namespace in v2 either way since v1 does `yarpc.inbounds.http` whereas v2 does/could do `yarpchttp.inbounds` or `yarpc.http.inbounds`, so people could still use both